dotnet/Vaas/src/Vaas/Vaas.cs

```diff
@@ -216,14 +216,16 @@ public async Task<VaasVerdict> ForStreamAsync(
 
         var response = WaitForResponseAsync(verdictResponse.Guid);
         await UploadStream(stream, verdictResponse.Url, verdictResponse.UploadToken, cancellationToken);
-        
+
         return new VaasVerdict(await response);
     }
     
     private async Task UploadStream(Stream stream, string url, string token, CancellationToken cancellationToken)
     {
         using var requestContent = new StreamContent(stream);
         using var requestMessage = new HttpRequestMessage(HttpMethod.Put, url);
+        requestMessage.Version = HttpVersion.Version11;

```

Comment:
Isn't that the default anyway according to https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.defaultrequestversion?view=net-8.0

Reply:
It's set by default but in case there are lib changes, where they changed the version from 1.1 to 2.0, it's better to set it in the SDK. Also, it's more transparent for us.